src/main/java/frc/robot/Robot.java

```diff
@@ -95,28 +110,44 @@ public void teleopInit() {
 
     public Command switchCameras() {
         return new InstantCommand(() -> {
-            System.out.println("Camera 0" + camera0Active);
-            if (!camera0Active) {
-                videoSink.setSource(camera0);
-                camera0Active = !camera0Active;
+            System.out.println("Camera 0" + cameraBackActive);
+            if (!cameraBackActive) {
+                videoSink.setSource(cameraBack);
+                cameraBackActive = !cameraBackActive;
             } else {
-                videoSink.setSource(camera1);
-                camera0Active = !camera0Active;
+                videoSink.setSource(cameraFront);
+                cameraBackActive = !cameraBackActive;
             }
         });
     }
 
+    public Command goodFlip() {
+        CommandChain retValue = new CommandChain("Good Flip");
+        retValue.then(lift.goToHeight(LiftSubsystem.Heights.kLiftFlipHeight)).then(maflipulator.crappyFlip());
+        return retValue;
+    }
+
     public Command darkenCameras() {
         return new InstantCommand(() -> {
-            camera0.setBrightness(30);
-            camera1.setBrightness(30);
+            cameraBack.setBrightness(0);
+            cameraFront.setBrightness(0);
         });
     }
 
     public Command brightenCameras() {
         return new InstantCommand(() -> {
-            camera0.setBrightness(100);
-            camera1.setBrightness(100);
+            cameraBack.setExposureAuto();;
+            cameraFront.setExposureAuto();
         });
     }
+
+public void assignButtons() {
+    Robot.xBoxCoPilot.getButton(ButtonXboxController.XBoxButton.BUMPER_LEFT.get()).whenPressed(manipulator.openBeak());
```

Comment:
Not that it helps the line length much, but you don't need that `.get()` there, you can stop at `BUMPER_LEFT`

Reply:
Also you're already inside `Robot` so you can just say `xBoxCoPilot.getButton(...)`.

Reply:
done